README.md

```diff
@@ -1,63 +1,168 @@
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+4. To run a local node, open `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-### Sent token
+   The five supported networks are deployed: Moonbeam (ChainID: 2500), Avalanche (chainID: 2501), Fantom (chainID: 2502), Ethereum (chainID: 2503), and Polygon (chainID: 2504) on port 8500.
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
 
-### Deposit Address
+## Deploy and test each example
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+For each example, you can allow the defaults to run or enter:
+
+- `local` or `testnet` for both the test and the deploy steps, except for deposit-address and send-token, which run on testnet.
+
+- `source-chain` and `destination-chain`
+
+\[Editor: What's the correct way to identify each? by chain-ID? as a "string" or integer?]
+
+- variables such as `message`, `amount`, and `account`
+
+\[Editor: What is the expected syntax and data type: message: "message"?, amount: integer?, account: "string" or integer? What are the defaults for each?]
+
+\[Editor: In regard to this phrase: "All params are optional and have default values and can be ommited." I think the writer means it in the sense that it's optional for the user to enter a value bc there are defaults. In API, the convention is to call variables with default values "required". I've reworded this to avoid this ambiguity.]
+
+and run the local deploy and test.
 
 ### Call contract
 
-Run `node scripts/test examples/call-contract ${local|testnet} ${source-chain} ${destination-chain} ${message}` to relay a message from `source-chain` to `destination-chain`.
+This dapp relays a message from source-chain to destination-chain.
+
+1. To deploy the dapp, run::
+
+`node scripts/deploy examples/call-contract [<local|testnet>]`
+
+2. To test it, run:
+
+`node scripts/test examples/call-contract [<local|testnet>] [<source-chain] [<destination-chain>] [<message]`
```

Comment:
@ggutoski  This didn't work. I think it's the message syntax. What's the proper syntax. I tried " ' and none:
It deployed, but didn't test:
annemariejackson@ANNEs-MacBook-Pro axelar-local-gmp-examples % node scripts/deploy examples/call-contract local
Deploying ExecutableSample for Moonbeam.
Deploying ExecutableSample for Avalanche.
Deploying ExecutableSample for Fantom.
Deploying ExecutableSample for Ethereum.
Deploying ExecutableSample for Polygon.
Deployed ExecutableSample for Avalanche at 0x775C53cd1F4c36ac74Cb4Aa1a3CA1508e9C4Bd24.
Deployed ExecutableSample for Ethereum at 0x775C53cd1F4c36ac74Cb4Aa1a3CA1508e9C4Bd24.
Deployed ExecutableSample for Fantom at 0x775C53cd1F4c36ac74Cb4Aa1a3CA1508e9C4Bd24.
Deployed ExecutableSample for Moonbeam at 0x775C53cd1F4c36ac74Cb4Aa1a3CA1508e9C4Bd24.
Deployed ExecutableSample for Polygon at 0x775C53cd1F4c36ac74Cb4Aa1a3CA1508e9C4Bd24.
annemariejackson@ANNEs-MacBook-Pro axelar-local-gmp-examples % node scripts/test examples/call-contract local 2500 2501 'Hello Axelar!'
--- Initially ---
/Users/annemariejackson/axelar-local-gmp-examples/examples/call-contract/index.js:28
        console.log(`value at ${destination.name} is ${await destination.contract.value()}`)
                                            ^

TypeError: Cannot read properties of undefined (reading 'name')
    at print (/Users/annemariejackson/axelar-local-gmp-examples/examples/call-contract/index.js:28:45)
    at Object.test (/Users/annemariejackson/axelar-local-gmp-examples/examples/call-contract/index.js:37:11)
    at /Users/annemariejackson/axelar-local-gmp-examples/scripts/test.js:37:19
    at Object.<anonymous> (/Users/annemariejackson/axelar-local-gmp-examples/scripts/test.js:42:3)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
annemariejackson@ANNEs-MacBook-Pro axelar-local-gmp-examples % node scripts/test examples/call-contract local 2500 2501 "Hello Axelar!"
dquote> node scripts/test examples/call-contract local 2500 2501 Hello Axelar!  
dquote> "

Reply:
You ran
```
node scripts/test examples/call-contract local 2500 2501 'Hello Axelar!'
```
but `2500` and `2501` are not chain names.  (Where did you get these numbers?)  Instead you should use chain names from `Moonbeam`, `Avalanche`, `Fantom`, `Ethereum`, `Polygon`.

I think `'Hello Axelar!'` will work but you may wish to play it safe for a first test and use something without a space or special character such as `Annamarie`.

Reply:
@ggutoski Thanks. How to refer to the chains was one of my questions. I got the chain names and numbers from the deploy and guessed we would id them by number. The "dquote>" response threw me off. OK. I am further along. Thanks.

Reply:
The `dquote>` is a shell prompt.  It thinks you're entering a multiline quoted string.  It acts this way because the shell prompt treats `!` as a special character.  You need to use single quotes `'` instead of double quotes `"` for strings that have exclamation points in them.  Counter-intuitive, I know.  See https://unix.stackexchange.com/a/33340